mozilla-release/browser/app/profile/firefox.js

```diff
@@ -1272,7 +1272,8 @@ pref("services.sync.prefs.sync.browser.safebrowsing.malware.enabled", true);
 pref("services.sync.prefs.sync.browser.search.update", true);
 pref("services.sync.prefs.sync.browser.sessionstore.restore_on_demand", true);
 pref("services.sync.prefs.sync.browser.startup.homepage", true);
-pref("services.sync.prefs.sync.browser.startup.page", true);
+pref("services.sync.prefs.sync.browser.startup.restoreTabs", true);
+pref("services.sync.prefs.sync.browser.startup.addFreshTab", true);
```

Comment:
Technically it's possible, but will add more complexity to preference value handling, because we have 2 separate controls now. So, 2 separate values actually just reflect that those are independent preferences now.